src/test/java/uk/ac/ebi/eva/commons/models/converters/data/DBObjectToVariantSourceEntryConverterTest.java

```diff
@@ -90,40 +90,20 @@ public void setUp() {
     }
 
     @Test
-    public void testConvertToDataModelTypeWithoutStats() {
-        file.getSamplesData().clear(); // TODO Samples can't be tested easily, needs a running Mongo instance
-        List<String> sampleNames = new ArrayList<String>();
-
-        // Test with no stats converter provided
-        DBObjectToVariantSourceEntryConverter converter = new DBObjectToVariantSourceEntryConverter(
-                new DBObjectToSamplesConverter(sampleNames));
-        VariantSourceEntry converted = converter.convert(mongoFile);
-        assertEquals(file, converted);
-
-        // Test with a stats converter provided but no stats object
-        converter = new DBObjectToVariantSourceEntryConverter(
-                new DBObjectToSamplesConverter(sampleNames));
-        converted = converter.convert(mongoFile);
-        assertEquals(file, converted);
-    }
-
-    @Test
     public void testConvertToDataTypeWithoutStatsWithSampleIds() {
```

Comment:
Shouldn't this be named `testConvertToDataModelTypeWithoutStats`, given that there won't be sample IDs from now on?